src/Php84/bootstrap.php

```diff
@@ -15,6 +15,14 @@
     return;
 }
 
+if (function_exists('curl_version') && curl_version()['version'] >= 0x074200) { // libcurl >= 7.66.0
```

Comment:
We can turn this into a fast call in PHP 8.2, see https://github.com/php/php-src/pull/8720

```suggestion
if (defined('CURL_VERSION_HTTP3') || PHP_VERSION_ID < 80200 && function_exists('curl_version') && curl_version()['version'] >= 0x074200)) { // libcurl >= 7.66.0
```